### PR TITLE
Trigger bit combiner secondary input

### PR DESCRIPTION
**Pull request by jhgoh**
TriggerBitCombiner accepts secondary InputTag to run on various use cases.
Now the module is EDFilter. User can turn on/off event filtering by option.

### CODE REVIEW

**Comment by jhgoh** [comment]
Test to be done.

File: CatAnalyzer/test/run_TtbarDiLeptonAnalyzer_cfg.py [hunk 1]
```diff
@@ -47,7 +47,7 @@
         jetConeSize = cms.double(0.4),
     )
 
-process.filterRECO = cms.EDProducer("CATTriggerBitCombiner",
+process.filterRECO = cms.EDFilter("CATTriggerBitCombiner",
     triggerResults = cms.InputTag("TriggerResults::CAT"),
```
Comment (jshlee) [inline CatAnalyzer/test/run_TtbarDiLeptonAnalyzer_cfg.py]: can u change this to TriggerResults::PAT and have secondaryTriggerResults as TriggerResults::RECO

**Comment by jhgoh** [comment]
Done.

**Comment by jhgoh** [comment]
@tt8888tt could you confirm this is running properly?

**Comment by jshlee** [comment]
should be 
    triggerResults = cms.InputTag("TriggerResults::PAT"),
    secondaryTriggerResults = cms.InputTag("TriggerResults::RECO"),

**Comment by jshlee** [comment]
tested and works

**Comment by jhgoh** [comment]
Done.

**Pull request merged**